functions/activate-batch-background.js

```diff
@@ -92,6 +112,13 @@ async function handle(data, db, contract) {
             prizes: '[]'
           }
         await db.put('batches', batchItem)
+
+        const accountUpdateBadluckCount = {
+          address: a,
+          ...addressBadluckCount[a]
+        }
+
```

Comment:
This could be a non-existent account (First time user that hasn't opened box yet), so it needs to be initialised with all the right values fragments etc like in open-box